src/patternfly/components/Accordion/accordion.scss

```diff
@@ -85,6 +87,11 @@
 
   background-color: var(--#{$accordion}--BackgroundColor);
 
+  &.pf-m-start {
+    --#{$accordion}__toggle--JustifyContent: start;
+    --#{$accordion}__toggle--ColumnGap: var(--#{$pf-global}--spacer--md);
```

Comment:
Good call on this column gap. Looks like right now the thing that creates space between the text and icon is a `max-width` on the text. I'm not sure why that's there, and a margin or something wasn't used instead? (Here's the original PR for this change, I didn't see anything that stood out as to why -https://github.com/patternfly/patternfly/pull/1357)

Not sure if it makes any difference, but now that you have the column gap, you can probably set `--pf-v5-c-accordion__toggle-text--MaxWidth: 0;` in the the start aligned selector. I don't think you need a local component var for that since it isn't something we'd want anyone to change.

You may also leave a comment (or create an issue) to apply this column-gap to the default toggle and remove the max-width.

Reply:
https://github.com/patternfly/patternfly/issues/6013